File: client/bcephal-client-model/Security/UserInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Bcephal.Models.Security
{
    public class UserInfo
    {
        public string Login { get; set; }
        public string FirstName { get; set; }
        public string Name { get; set; }
        public string DefaultLanguage { get; set; }

        public string FullDescription { get { return Login + " (" + FirstName + " " + Name + ")"; } }

    }
}

File: server/bcephal-gateway-service/src/main/java/com/moriset/bcephal/gateway/domain/UserInfo.java
/**
 * 
 */
package com.moriset.bcephal.gateway.domain;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * @author Joseph Wambo
 *
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class UserInfo {

	private String login;
	private String firstName;
	private String name;
	private String defaultLanguage;
	
}
